multipy/runtime/interpreter/interpreter_impl.cpp

```diff
@@ -457,5 +473,20 @@ torch::deploy::InterpreterImpl*
 newInterpreterImpl(
     const std::vector<std::string>& extra_python_paths,
     const std::vector<std::string>& plugin_paths) {
-  return new ConcreteInterpreterImpl(extra_python_paths, plugin_paths);
+  ConcreteInterpreterImplConstructorCommon(extra_python_paths, plugin_paths);
```

Comment:
Is there a reason we still have this. It looks like we can use the `ConcreteInterpreterImpl` constructor to do the same thing. We shouldn't add the extra complexity until it becomes evidently neccessary.  Let's get rid of `ConcreteInterpreterImplConstructorCommon`

Reply:
This is to get the logic out into a 'default' visibility space.
The nontrivial lints were triggered by the refactor, but yeah agree that we should probably fix them for BE.